<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>covarianceRawDialog</class>
 <widget class="QDialog" name="covarianceRawDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="labelRawFile">
       <property name="text">
        <string>Raw file to use:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditRawFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBrowse">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelBeginTime">
     <property name="text">
      <string>Beginning of time interval (seconds from start):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxStartTime">
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string> s</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelEndTime">
     <property name="text">
      <string>End of time interval (seconds from start):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxEndTime">
     <property name="suffix">
      <string> s</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxRejection">
     <property name="title">
      <string>Rejection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QFrame" name="frameRejectionPeakToPeak">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelRejections">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Reject chunks of data based on peak to peak amplitude:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_30">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_31">
               <item>
                <widget class="QCheckBox" name="checkBoxGradReject">
                 <property name="text">
                  <string>grad.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBoxGradReject">
                 <property name="prefix">
                  <string/>
                 </property>
                 <property name="suffix">
                  <string> fT/cm</string>
                 </property>
                 <property name="maximum">
                  <double>1000000000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>4000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_32">
               <item>
                <widget class="QCheckBox" name="checkBoxEEGReject">
                 <property name="text">
                  <string>EEG</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBoxEEGReject">
                 <property name="suffix">
                  <string> uV</string>
                 </property>
                 <property name="maximum">
                  <double>1000000000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>40.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_33">
               <item>
                <widget class="QCheckBox" name="checkBoxMagReject">
                 <property name="text">
                  <string>mag.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBoxMagReject">
                 <property name="suffix">
                  <string> fT</string>
                 </property>
                 <property name="maximum">
                  <double>1000000000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>4000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_34">
               <item>
                <widget class="QCheckBox" name="checkBoxEOGReject">
                 <property name="text">
                  <string>EOG</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBoxEOGReject">
                 <property name="suffix">
                  <string> uV</string>
                 </property>
                 <property name="maximum">
                  <double>1000000000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>250.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelChunkLength">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Chunk length for rejection (tstep):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBoxChunkLength">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> s</string>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QFrame" name="frameRejectionFlatness">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelRejectionsFlatness">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Reject chunks of data based on signal flatness:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_40">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_41">
               <item>
                <widget class="QCheckBox" name="checkBoxFlatGrad">
                 <property name="text">
                  <string>grad.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBoxFlatGrad">
                 <property name="prefix">
                  <string/>
                 </property>
                 <property name="suffix">
                  <string/>
                 </property>
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.010000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_42">
               <item>
                <widget class="QCheckBox" name="checkBoxFlatEeg">
                 <property name="text">
                  <string>EEG</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBoxFlatEEG">
                 <property name="suffix">
                  <string/>
                 </property>
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.010000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QCheckBox" name="checkBoxFlatECG">
                 <property name="text">
                  <string>ECG</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBoxFlatECG">
                 <property name="suffix">
                  <string/>
                 </property>
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.010000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_43">
               <item>
                <widget class="QCheckBox" name="checkBoxFlatMag">
                 <property name="text">
                  <string>mag.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBoxFlatMag">
                 <property name="suffix">
                  <string/>
                 </property>
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.010000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_44">
               <item>
                <widget class="QCheckBox" name="checkBoxFlatEOG">
                 <property name="text">
                  <string>EOG</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBoxFlatEOG">
                 <property name="suffix">
                  <string/>
                 </property>
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.010000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxRejection">
        <property name="text">
         <string>Reject signals based on parameters below</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxIncludeChannels">
     <property name="title">
      <string>Include channels:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButtonIncludeAll">
          <property name="text">
           <string>All except bad</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupIncludeChannels</string>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="radioButtonIncludeList">
            <property name="text">
             <string>Only the following:</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupIncludeChannels</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEditIncludeChannelList">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="plainText">
             <string>***example of input here***Preferably also possibility to choose channels based on region***</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>covarianceRawDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>591</x>
     <y>685</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>covarianceRawDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>591</x>
     <y>685</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonIncludeList</sender>
   <signal>toggled(bool)</signal>
   <receiver>plainTextEditIncludeChannelList</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRejection</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelChunkLength</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRejection</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBoxChunkLength</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRejection</sender>
   <signal>toggled(bool)</signal>
   <receiver>frameRejectionPeakToPeak</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRejection</sender>
   <signal>toggled(bool)</signal>
   <receiver>frameRejectionFlatness</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroupIncludeChannels"/>
 </buttongroups>
</ui>
